docs/readmes/lte/dev_notes.md

```diff
@@ -48,6 +48,72 @@ file is generated by mapping the configuration values from `gateway.mconfig` and
 `/etc/systemd/system` directory of the VM at the time of provisioning. You need
 to manually sync changes to these files between guest and host OS.
 
+- Systemd configuration files contain interdependencies of Magma services regarding the start, stop, and restart of services. Therefore, a systemd configuration contains several options to specify these dependencies.
+- The options `Before` and `After` define the order in which services are started and stopped. If a service gets started, the services listed in `Before` and `After` will also be started. If one of the other services gets stopped or its start fails, the service will be stopped. The following dependency graph depicts this order for Magma services.
+
```

Comment:
Can we insert a short an pregnant "How to read an arrow" here. Similar to the ones in https://github.com/magma/magma/issues/14055#issuecomment-1282184443.
E.g. Read `ConnectionD → MME` as "ConnectionD has a Before=MME configured", or "MME has a After=ConnectionD" configured. Double check, that I have given that in the right direction in this example here.